src/test/project-manager.test.ts

```diff
@@ -31,4 +32,24 @@ describe('ProjectManager', () => {
 			assert.equal(projectManager.getConfiguration('subdirectory-with-tsconfig/src/dummy.ts').getPackageName(), 'package-name-2');
 		});
 	});
+	describe('ensureReferencedFiles', () => {
+		before(async () => {
+			memfs = new InMemoryFileSystem('/');
+			const localfs = new MapFileSystem(new Map([
+				['file:///package.json', '{"name": "package-name-1"}'],
+				['file:///node_modules/somelib/index.js', '/// <reference path="./pathref.d.ts"/>\n/// <reference types="node"/>'],
+				['file:///node_modules/somelib/pathref.d.ts', ''],
+				['file:///node_modules/@types/node/index.d.ts', ''],
+				['file:///src/dummy.ts', 'import * as somelib from "somelib";']
+			]));
+			const updater = new FileSystemUpdater(localfs, memfs);
+			projectManager = new ProjectManager('/', memfs, updater, true);
+			await projectManager.ensureReferencedFiles('file:///src/dummy.ts').toPromise();
```

Comment:
This is the unit under test, it should be called in the test, not the setup code.